src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const peopleList = document.querySelectorAll('li');
+
+const getPersonSalary = person => {
+  return Number(person.dataset.salary.replace(/[$,]/g, ''));
+};
+
+function sortList(list) {
+  [...list].sort((personA, personB) => {
+    const personASalary = getPersonSalary(personA);
+    const personBSalary = getPersonSalary(personB);
+
+    if (personASalary > personBSalary) {
+      personB.before(personA);
+    }
+
+    return personASalary - personBSalary;
+  });
+}
+
+function getEmployees(list) {
+  const employeesList = [...list].map(person => {
```

Comment:
```suggestion
  const employeesList = list.map(person => {
```
Метод map не мутуючий, тобі не потрібно створювати копію